tasks/section1.yml

```diff
@@ -1252,15 +1252,20 @@ print $NF }'"
       - rule_1.7.1.4
 
 - name: "NOTSCORED | 1.7.1.4 | PATCH | Ensure permissions on /etc/motd are configured"
-  command: /bin/true
+  file:
+      dest=/etc/motd
+      state=file
+      owner=root
+      group=root
+      mode=0644
   tags:
       - level1
       - level2
       - patch
       - rule_1.7.1.4
 
 - name: "SCORED | 1.7.1.5 | AUDIT | Ensure permissions on /etc/issue are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/issue  | egrep "644 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*